docs/source/cli.rst

```diff
@@ -2,7 +2,7 @@ Command Line Interface
 ======================
 
 iBridges also has a Command Line Interface (CLI). The CLI provides an even more simplified
-interface for uploading, downloading and synchronizing. It does not provide (nor intends to) all the features
+interface for uploading, downloading and synchronising. It does not provide (nor intends to) all the features
```

Comment:
I'm generally using American spelling, wouldn't it be "synchronizing"?

Reply:
Yes .. done, though the link will only work once we moved the rtd to main.

Reply:
I am torn between two spellcheckers ... one for markdown and for jupyter notebooks ... I think most of what I wrote now is BE. Shall I switch to AE?